### ISSUE TITLE
[HL2] Legacy enemy flinching

### ISSUE THREAD

**Issue by DenAkimov** [ValveSoftware/Source-1-Games#6627]
In early versions of the game enemies (metrocops and combine soldiers) were a lot more reactive to being damaged by the player, which seems to have been lost with time. 

Current build:

https://github.com/user-attachments/assets/5569bf0e-0879-4324-a252-9bf9a2193c82

Archived 2004 build:

https://github.com/user-attachments/assets/5882dfcd-ab83-41de-8743-a55947addf46

**Comment by Landate** [comment]
This need fixed!

**Comment by Zeekoff** [comment]
I really hope this gets patched to be like the 2004 build.

**Comment by DarthRazdor** [comment]
I agree, they SHOULD fix this. Bodies only react once every 3-4 seconds. Broken since 2009 update. They ignore your shots most of the time and shoot back. In the old version, you could prevent enemies from firing back.
Here's another example.

ORIGINAL
![HalfLifeSource2024 12 03-14 00 43 14-ezgif com-optimize](https://github.com/user-attachments/assets/3cda4d1d-8d7d-4838-a13f-a5044222b6a7)

CURRENT
![Half-Life22024 12 03-13 59 31 13-ezgif com-optimize](https://github.com/user-attachments/assets/ac95806a-11bf-45e3-9747-281e7606f246)
